### ISSUE TITLE
ShowSignIn function doesnt show the login view properly 

### ISSUE THREAD

**Issue by ponchonov** [aws-amplify/aws-sdk-ios#2029]
**Describe the bug**
The ASWAuthUI is integrated using Cocoapods and when try to show the sign in view using ShowSignIn doesnt show the form to login, only shows the Amplify logo, in the console shows: 

`Cognito[12008:675365] [TableView] Warning once only: UITableView was told to layout its visible cells and other contents without being in the view hierarchy (the table view or one of its superviews has not been added to a window). This may cause bugs by forcing views inside the table view to load and perform layout without accurate information (e.g. table view bounds, trait collection, layout margins, safe area insets, etc), and will also cause unnecessary performance overhead due to extra layout passes. Make a symbolic breakpoint at UITableViewAlertForLayoutOutsideViewHierarchy to catch this in the debugger and see what caused this to occur, so you can avoid this action altogether if possible, or defer it until the table view has been added to a window. Table view: <UITableView: 0x7fdb44039e00; frame = (4 4; 271 142); clipsToBounds = YES; autoresize = RM+BM; gestureRecognizers = <NSArray: 0x600003608b10>; layer = <CALayer: 0x6000038368a0>; contentOffset: {0, 0}; contentSize: {271, 0}; adjustedContentInset: {0, 0, 0, 0}; dataSource: (null)>
`

**To Reproduce**

In the viewDidLoad call the 'ShowSignIn' function

**Expected behavior**

Show the Sign In view for login
**Screenshots**

![Screenshot](https://p195.p4.n0.cdn.getcloudapp.com/items/7KuxWLbv/Image+2019-11-06+at+12.48.57+PM.png?v=6fd926934e9fa8363c6387500dc26937)

**Environment(please complete the following information):**
 - SDK Version: [2.12.1]
 - Dependency Manager: [Cocoapods]
 - Swift Version : [5.0]

**Device Information (please complete the following information):**
 - Device: [iPhone11, Simulator]

**Comment by wooj2** [comment]
Hi,
I tried to reproduce your issue using CocoaPods, SDK version 2.12.1, and an iPhone 11 Simulator, but was unfortunately unable to reproduce your issue.

I generally followed instructions located here:
https://aws-amplify.github.io/docs/ios/authentication

Are you able to share any additional data or configuration specifics you are using in your application that might be causing this issue?

Thanks

**Comment by KamikazeZirou** [comment]
I also encountered the same behavior.
The following warning log was output.
```
Found UserPool configuration in awsconfiguration.json but could not find dependencies. Skipping rendering in AuthUI
```

In my case,  the problem was solved by adding the following description to Podfile.
```
pod 'AWSUserPoolsSignIn', '~> 2.12.0'
```
(See https://aws-amplify.github.io/docs/ios/authentication#install-dependencies)


I use Hosted UI for Auth0 at first, but changed it to use Drop In UI on the way.
So I forgot this description of Podfile.
